draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1608,10 +1608,12 @@ compute the cookie-string from a cookie store and a request-uri:
 4.  Serialize the cookie-list into a cookie-string by processing each cookie
     in the cookie-list in order:
 
-    1.  Output the cookie's name, the %x3D ("=") character, and the cookie's
-        value.
+    1.  If the cookies' name is not empty, output the cookie's name followed by
+        the %x3D ("=") character.
 
-    2.  If there is an unprocessed cookie in the cookie-list, output the
+    2.  If the cookies' value is not empty, output the cookie's value.
```

Comment:
Maybe just `2.  Output the cookie's value.` would be enough because if it's empty then nothing is output anyway.

Reply:
Thanks! I think I vaguely prefer being explicit about how an empty value would be serialized (e.g. not at all)... probably doesn't make much of a difference, but keeping it the same as the 1 above seems reasonable to me.